elasticdownload/src/main/java/is/arontibo/library/ElasticDownloadView.java

```diff
@@ -107,5 +107,30 @@ public void onEnterAnimationFinished() {
 
         // Do further actions if necessary
     }
+    
+
+    /**
+     * Set the background color of the Elastic Download View
+     * @param passedColor int Color Color to set the background
+     */
+    public void setBackgroundColor(int passedColor) {
+        this.mBackgroundColor = passedColor;
+        this.mProgressDownloadView.setBackgroundColor(mBackgroundColor);
+    }
+
+    /**
+     * Overloaded method, takes in a String color to parse
+     * @param passedColor String of a color hex color (IE: #fd5c79) that can be parsed by
+     *                    Color.parseColor(string)
+     */
+    public void setBackgroundColor(String passedColor) {
+        if(passedColor == null){
```

Comment:
this `if` needs a space between itself and the parenthesis (like the rest of the code).